.github/workflows/jdbc-tests-single-db-mode.yml

```diff
@@ -0,0 +1,177 @@
+          ~/${{env.INSTALL_DIR}}/bin/psql -v ON_ERROR_STOP=1 -d jdbc_testdb -U runner -p 5433 -c "CREATE SUBSCRIPTION my_sub CONNECTION 'host=localhost port=5432 user=jdbc_user dbname=jdbc_testdb password=12345678' PUBLICATION my_pub;"
+
+      - name: Run Replication Tests
+        id: replication
+        if: always() && steps.setup-pub-sub.outcome == 'success'
+        timeout-minutes: 60
+        uses: ./.github/composite-actions/run-jdbc-tests
+        with:
+          input_dir: 'replication'
+
+      - name: Cleanup babelfish database
+        id: cleanup
+        if: always() && steps.replication.outcome == 'success'
+        run: |
+          sudo ~/psql/bin/psql -d postgres -U runner -v user="jdbc_user" -v db="jdbc_testdb" -f .github/scripts/cleanup_babelfish_database.sql
+
+      - name: Upload Log
+        if: always() && (steps.jdbc.outcome == 'failure' || steps.replication.outcome == 'failure')
+        uses: actions/upload-artifact@v2
+        with:
+          name: postgres-log-jdbc
+          path: |
+            ~/psql/data/logfile
+            ~/psql/data_5433/logfile
+
+      # The test summary files contain paths with ':' characters, which is not allowed with the upload-artifact actions
+      - name: Rename Test Summary Files
+        id: test-file-rename
+        if: always() && (steps.jdbc.outcome == 'failure' || steps.replication.outcome == 'failure')
+        run: |
+          cd test/JDBC/Info
+          timestamp=`ls -Art | tail -n 1`
+          cd $timestamp
+          mv $timestamp.diff ../output-diff.diff
+          mv "$timestamp"_runSummary.log ../run-summary.log
+          cd ..
+          # get the replication output diff as well if it is present
+          dir_count=`ls | wc -l`
+          if [[ $dir_count -eq 2 ]];then
+            timestamp=`ls -rt | tail -n 2 | sort -r | tail -n 1`
+            cd $timestamp
+            mv $timestamp.diff ../replication-output-diff.diff
+            mv "$timestamp"_runSummary.log ../replication-run-summary.log
+
+      - name: Upload Run Summary 
+        if: always() && steps.test-file-rename.outcome == 'success'
+        uses: actions/upload-artifact@v2
+        with:
+          name: run-summary.log
+          path: |
+            test/JDBC/Info/run-summary.log
+            test/JDBC/Info/replication-run-summary.log
+
+      - name: Upload Output Diff
+        if: always() && (steps.jdbc.outcome == 'failure' || steps.replication.outcome == 'failure')
+        uses: actions/upload-artifact@v2
+        with:
+          name: jdbc-output-diff.diff
+          path: |
+            test/JDBC/Info/output-diff.diff
+            test/JDBC/Info/replication-output-diff.diff
+
+      - name: Check and upload coredumps
+        if: always() && (steps.jdbc.outcome == 'failure' || steps.replication.outcome == 'failure')
+        uses: ./.github/composite-actions/upload-coredump
+
+      - name: Generate Code Coverage
```

Comment:
If we want to run code-coverage for single-db, then we should update the pr-code-coverage which will invoke this workflow on [workflow_call]. Or else we should remove these.

Reply:
removed it.